### PR TITLE
New Config Method .save()

### PR DESCRIPTION
**Pull request by Moumouls**
About the [PR Parse.Config.save()](https://github.com/parse-community/Parse-SDK-JS/pull/684)

### CODE REVIEW

File: _includes/js/config.md [hunk 1]
```diff
@@ -13,7 +13,26 @@ Parse.Config.get().then(function(config) {
...
+`ParseConfig` can be managed through the SDK when a `Master Key` is provided. You can save new parameters and if you save already existing parameters they will be automatically updated.
+After a successfully `.save()` it return the new updated `ParseConfig` and `Parse.Config.current()` is up to date too.
+
+```javascript
+Parse.Config.save({
+	welcomeMesssage : "Welcome to Parse",
+	ageOfParse : 3,
+	tags : ["parse","sdk","js"]
+}).then(function(config) {
+  console.log("Cool! Config was saved ans fetched from the server.");
```
Comment (flovilmart) [inline _includes/js/config.md]: typo:
```suggestion
  console.log("Cool! Config was saved and fetched from the server.");
```

File: _includes/js/config.md [hunk 2]
```diff
@@ -13,7 +13,26 @@ Parse.Config.get().then(function(config) {
...
+  
+  var welcomeMessage = config.get("welcomeMessage");
```
Comment (flovilmart) [inline _includes/js/config.md]: prefer const, it's 2018 :)

```suggestion
  const welcomeMessage = config.get("welcomeMessage");
```

File: _includes/js/config.md [hunk 3]
```diff
@@ -13,7 +13,26 @@ Parse.Config.get().then(function(config) {
...
 
+`ParseConfig` can be managed through the SDK when a `Master Key` is provided. You can save new parameters and if you save already existing parameters they will be automatically updated.
+After a successfully `.save()` it return the new updated `ParseConfig` and `Parse.Config.current()` is up to date too.
+
+```javascript
+Parse.Config.save({
+	welcomeMesssage : "Welcome to Parse",
+	ageOfParse : 3,
+	tags : ["parse","sdk","js"]
+}).then(function(config) {
```
Comment (flovilmart) [inline _includes/js/config.md:26]: prefer arrow functions
```suggestion
}).then((config) => {
```

File: _includes/js/config.md [hunk 4]
```diff
@@ -13,7 +13,26 @@ Parse.Config.get().then(function(config) {
   // Something went wrong (e.g. request timed out)
 });
 ```
+## Save a Config
 
+`ParseConfig` can be managed through the SDK when a `Master Key` is provided. You can save new parameters and if you save already existing parameters they will be automatically updated.
```
Comment (flovilmart) [inline _includes/js/config.md]: > when a `Master Key` is provided

ie: only in nodejs :)

**Review by flovilmart — CHANGES_REQUESTED** [review]
Small nits otherwise that's cool!

File: _includes/js/config.md (hunk 4 above)
Comment (Moumouls) [reply _includes/js/config.md]: it doesn't works in a browser with a masterkey ?

Comment (flovilmart) [reply _includes/js/config.md]: Master key should NEVER be exposed in the browser! and the JS SDK doesn’t support passing a third argument.

**Review by flovilmart — APPROVED** [review]
Very nice! Thanks

**Comment by Moumouls** [comment]
I just noticed that that i forgot to transform `function(){}` to `() => {}`

**Comment by flovilmart** [comment]
Not a big deal :)

**Pull request merged**